.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -1314,4 +1314,43 @@ stages:
       VHD_BUILD_ID: $(Build.BuildId)
       LOCATION: $(PACKER_BUILD_LOCATION)
     jobs:
-      - template: ./templates/e2e-template.yaml
+      - template: ./templates/e2e-template.yaml  
+  - stage: Create_Production_VMs
+    dependsOn: build
+    condition: eq(variables.Create_Production_VMS, 'true'), 
+    steps:
+    - task: DownloadPipelineArtifact@2
+      displayName: Download VHD info artifact
+      inputs:
+        artifact: 'vhd-info'
+        targetPath: '$(Pipeline.Workspace)/downloaded-artifacts' # Download the vhd-publishing-info.json into to the workspace
+    - Job: Create_Production_VMs
+      pool:
+        name: nodesigtest-pool # todo (alburgess) check if this is correct
```

Comment:
we shouldn't hardcode pool names (instead reference a variable such as `$(POOL_NAME)` which we can configure on the pipeline level